.gitignore

```diff
@@ -5,8 +5,12 @@ target/
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
+# Private key files
+*.pem
+
 # bee-node
 bee-node/config.toml
+bee-node/identity.toml
```

Comment:
I guess that line is wrong. It should be `bee-node/identity.pem`, right? Do we need the general exclusion of `.pem` files (above) then?